### DISCUSSION TITLE
Change host port?

### DISCUSSION THREAD

**Discussion by vicitooo** [oobabooga/text-generation-webui#1946] [Q&A] [answered]
How can I change instead of starting on 127.0.0.0  to start on 0.0.0.0 ? I know how to change the : PORT after that, but not the host ?

**Comment by vicitooo** [comment, marked as answer]
In server.py 

    # Launch the interface
    shared.gradio['interface'].queue()
    if shared.args.listen:
        shared.gradio['interface'].launch(prevent_thread_lock=True, share=shared.args.share, server_name='0.0.0.0',
                                          server_port=shared.args.listen_port, inbrowser=shared.args.auto_launch,
                                          auth=auth)
    else:
        shared.gradio['interface'].launch(prevent_thread_lock=True, share=shared.args.share, server_name='0.0.0.0',
                                          server_port=shared.args.listen_port, inbrowser=shared.args.auto_launch,
                                          auth=auth)


Whatever you change 0.0.0.0 to it will run on it.

**Reply by vicitooo** [reply]
I asked I answered I commented and liked my comment.

**Reply by adavad** [reply]
You're welcome.
